Plugins/Skinning/src/SkinningComponent.cpp

```diff
@@ -1,65 +1,67 @@
 #include <SkinningComponent.hpp>
 
-#include <Core/Animation/Pose/PoseOperation.hpp>
 #include <Core/Geometry/Normal/Normal.hpp>
+#include <Core/Animation/Pose/PoseOperation.hpp>
 
 #include <Core/Animation/Skinning/DualQuaternionSkinning.hpp>
 #include <Core/Animation/Skinning/LinearBlendSkinning.hpp>
 #include <Core/Animation/Skinning/RotationCenterSkinning.hpp>
 
-using Ra::Core::DualQuaternion;
+#include "PositionBasedSkinning/PBS.hpp"
+
+
 using Ra::Core::Quaternion;
+using Ra::Core::DualQuaternion;
 
 using Ra::Core::TriangleMesh;
+using Ra::Core::Animation::Skeleton;
 using Ra::Core::Animation::Pose;
 using Ra::Core::Animation::RefPose;
-using Ra::Core::Animation::Skeleton;
 using Ra::Core::Animation::WeightMatrix;
 
+//typedef Ra::Core::Animation::Handle::SpaceType SpaceType;
 using SpaceType = Ra::Core::Animation::Handle::SpaceType;
 
-using Ra::Core::Skinning::FrameData;
 using Ra::Core::Skinning::RefData;
+using Ra::Core::Skinning::FrameData;
 
 using Ra::Engine::ComponentMessenger;
```

Comment:
please reorder type use and put the type alias after or before but not within it

Reply:
The uses and alias have been reordered.